tools/split_file_to_collection/test-data/make_test_data.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
@caleb-easterly do you know that planemo will generate you test data with `--update_test_data`?

Reply:
I didn't! that's a good tip, thanks